lib/cc/cli/test.rb

```diff
@@ -68,6 +69,12 @@ def initialize(line, line_text)
     end
 
     class Test < Command
+      ARGUMENT_LIST = "<engine_name>".freeze
+      SHORT_HELP = "Test engine".freeze
+      HELP = "#{SHORT_HELP}\n" \
+        "\n"\
```

Comment:
Could we add a line here that is something like "Validate that an engine is behaving correctly"